apps/console/src/features/connections/components/edit/settings/authenticator-settings.tsx

```diff
@@ -104,7 +104,7 @@ interface IdentityProviderSettingsPropsInterface extends TestableComponentInterf
 
 const OIDC_CLIENT_ID_SECRET_MAX_LENGTH: number = 100;
 const URL_MAX_LENGTH: number = 2048;
-const AUTHORIZED_REDIRECT_URL: string = "callbackUrl";
+const AUTHORIZED_REDIRECT_URLS: string[] = [ "callbackUrl", "callBackUrl" ];
```

Comment:
Shall we add a comment on keeping this as an array?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/5021/commits/87945765b210a71bd6c1a7043887384b3391d5d5